test/e2e/setup.go

```diff
@@ -23,6 +26,8 @@ type ClientSet struct {
 	Operations        redhatopenshift.OperationsClient
 	Kubernetes        kubernetes.Interface
 	MachineAPI        machineapiclient.Interface
+	VirtualMachines   mgmtcompute.VirtualMachinesClient
```

Comment:
keep all the azure client interfaces together and all the kubernetes client interfaces together